okapi-core/src/main/java/org/folio/okapi/managers/EnvManager.java

```diff
@@ -86,6 +86,6 @@ public EnvManager(EnvStore s) {
   }
 
   Future<Void> remove(String name) {
-    return envMap.removeNotFound(name);
+    return envMap.removeNotFound(name).compose(res -> envStore.delete(name).mapEmpty());
```

Comment:
There should be a unit test that fails if this fix were not applied.

Reply:
Saw this one coming :-)

Reply:
test added